src/components/lightbox/CaptionItem.tsx

```diff
@@ -0,0 +1,73 @@
+import { Button, Group, Stack, Textarea, Title } from "@mantine/core";
+
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import "react-virtualized/styles.css";
+import { Note } from "tabler-icons-react";
+import { useAppDispatch, useAppSelector } from "../../store/store";
+import { generatePhotoIm2txtCaption, editPhoto } from "../../actions/photosActions";
+
+type Props = {
+  isPublic: boolean;
+  photoDetail: any;
+  dispatch: any;
+};
+
+function getCaption(props: Props) {
+  if (props.photoDetail.captions_json.user != undefined) {
+    return props.photoDetail.captions_json.user;
+  } else {
+    return props.photoDetail.captions_json.im2txt;
+  }
+}
+
+export function CaptionItem(props: Props) {
+  const { generatingCaptionIm2txt } = useAppSelector(store => store.photos);
+  const { photoDetail, isPublic, dispatch } = props;
+  const { t } = useTranslation();
+
+  const onSubmit = () => {
+    const differentJson = { caption: getCaption(props) };
+    props.dispatch(editPhoto(props.photoDetail.image_hash, differentJson));
+  };
+
+  return (
+    <Stack>
+      <Group>
+        <Note></Note>
+        <Title order={4}>{t("lightbox.sidebar.caption")}</Title>
+      </Group>
+      {false && getCaption(props)}
+      <Stack>
+        <Textarea
```

Comment:
Why is this always disabled regardless of the state of `isPublic` ?

Reply:
I thought it  was about ```{false && getCaption(props)}```, which is never actually rendered 😅

Reply:
> have you tried putting `false` into the `disabled` attribute

Yes. I've also tried removing the `disabled` property completely and it is still disabled/locked from editing.

Reply:
> I thought it was about `{false && getCaption(props)}`, which is never actually rendered 😅

@derneuere I was wondering about that line too. Do you think it's safe to remove?

Reply:
If I remove the value parameter then it becomes editable.

Reply:
@polaroidkidd yes, that's correct. When i remove the `value` parameter, then the `Textarea` element becomes editable.

Reply:
mantime just released its v5. Perhaps it's resolved in that version. 

Otherwise I would suggest to conditionally add the attribute (even though I'm personally not a huge fan of the syntax, but it would solve the issue.

Reply:
I think we just have to use it as a controlled element: https://mantine.dev/core/textarea/#controlled

Add a state, which has as a default the generated value and then implement it as described.